src/main/java/org/veupathdb/service/eda/ds/metadata/AppsMetadata.java

```diff
@@ -71,20 +71,24 @@ public class AppsMetadata {
           viz("scatterplot", new BetaDivScatterplotPlugin())),
       app("distributions", "Distributions", "null",
           "Plot simple distributions for any continuous variable, including metadata (e.g. age, height, etc.) or microbial assay results.",
-          List.of(MICROBIOME_PROJECT),
+          List.of(MICROBIOME_PROJECT, CLINEPI_PROJECT, ALLCLINEPI_PROJECT),
```

Comment:
I think AllClinEpiDB is no more; we can probably remove at these locations.

Reply:
this was intended for a different branch, pushed erroneously as i forgot i had done this.. fixed now.